portfolio/src/main/java/com/google/sps/servlets/FormServlet.java

```diff
@@ -51,9 +51,22 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
     // Retrieve the maximum quantity of comments to display
-    String radioValueString = request.getParameter("value");
-    int max = Integer.parseInt(radioValueString);
+    String radioValueString = request.getParameter("quantity");
 
+    int max = 0; 
+
+    // Ensure quantity parameter is an integer
+    try {
+      max = Integer.parseInt(radioValueString);
+    } catch (Exception e) {
```

Comment:
Try to avoid catching generic `Exception`, this one here is a `NumberFormatException` I believe.

Reply:
adjusted